package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "polymer-ide",
+  "description": "Provides linting, autocompletion, and more for web components.",
+  "author": "Polymer Authors",
+  "license": "MIT",
+  "version": "0.1.2",
```

Comment:
Was this ever actually published? If so, this is a at least a patch change because it now has a new dependency